conf/.env

```diff
@@ -0,0 +1,4 @@
+SHIORI_WEBROOT=__PATH__
+SHIORI_PORT__PORT__
```

Comment:
I think you forgot the `=` ^w^
`SHIORI_PORT=__PORT__`

Reply:
Yes, thanks